test/opt/convert_relaxed_to_half_test.cpp

```diff
@@ -1222,6 +1222,147 @@ OpFunctionEnd
                                            defs_after + func_after, true, true);
 }
 
+TEST_F(ConvertToHalfTest, ConvertToHalfWithClosure) {
```

Comment:
Please use `SinglePassRunAndMatch` for tests as much as possible.  When done well, the make it obvious what the test is testing.